language/tools/move-mv-llvm-compiler/src/stackless/translate.rs

```diff
@@ -908,12 +922,19 @@ impl<'mm, 'up> FunctionContext<'mm, 'up> {
                 if let Some(s) = named_locals.get(&i) {
                     name = format!("local_{}__{}", i, s);
                 }
+                if mty.is_struct() {
+                    let (st, _) = mty
+                        .get_struct(self.get_global_env())
+                        .expect("must be struct");
+                    debug!(target: "sbc", "mty as struct {}", st.get_full_name_str());
+                }
                 let llval = self.module_cx.llvm_builder.build_alloca(llty, &name);
                 self.locals.push(Local {
                     mty: mty.instantiate(self.type_params),
                     llty,
                     llval,
                 });
+                self.llty_to_mty.insert(Box::new(llty), mty.clone());
```

Comment:
This is what I'm referring to-- I still see uses of the map here (I mean, in this branch which this PR is based on).  I don't see the commit where this was removed. Either that, or the UI is just not showing the latest updates.